sql_generators/active_users_aggregates_v3/templates/desktop_query.sql

```diff
@@ -73,21 +72,20 @@ SELECT
     active_hours_sum,
     first_seen_date
   ),
-  COUNT(DISTINCT IF(days_since_seen = 0, client_id, NULL)) AS daily_users,
-  COUNT(DISTINCT IF(days_since_seen < 7, client_id, NULL)) AS weekly_users,
-  COUNT(DISTINCT client_id) AS monthly_users,
-  COUNT(
-    DISTINCT IF(days_since_seen = 0 AND active_hours_sum > 0 AND uri_count > 0, client_id, NULL)
-  ) AS dau,
-  COUNT(DISTINCT IF(submission_date = first_seen_date, client_id, NULL)) AS new_profiles,
+  COUNTIF(is_daily_user) AS daily_users,
```

Comment:
Agree. Since this PR changes the upstream to the baseline_clients_last_seen views, these checks were updated in [PR-5324](https://github.com/mozilla/bigquery-etl/pull/5324) for completion. They also [check uniqueness](https://github.com/mozilla/bigquery-etl/blob/b60bc09aa63f97fd3d10e8ec0c6c651488099fe7/sql_generators/glean_usage/templates/baseline_clients_last_seen_v1.checks.sql#L17). Do you think that covers?

Reply:
Excellent!